src/components/footer.vue

```diff
@@ -62,17 +96,47 @@
 </style>
 
 <script>
+import axios from "axios";
 export default {
   data: () => ({
     email: "",
-    emailIsValid: false
+    snackbar: false,
+    message: "",
+    emailIsValid: false,
   }),
   methods: {
+    async submitEmail() {
+      await axios
+        .get(
+          "/.netlify/functions/mailchimp?email=" +
+            this.email
+        )
+        .then((response) => {
+          if (
+            response &&
+            response.statusCode &&
+            response.statusCode === 200 &&
+            response.body &&
+            response.body.status &&
+            response.body.status === "subscribed"
```

Comment:
```suggestion
            response.&&
            response.status === "subscribed"
```
I think it is not nested in the body property.